specification/columns/servicesubcategory.md

```diff
@@ -0,0 +1,102 @@
+|                           | Data Processing                   | Integration and transformation tasks to prepare data for analysis.                                                                                                                                     |
+|                           | Search                            | Discovery of information by indexing and retrieving data from various sources.                                                                                                                         |
+|                           | Streaming Analytics               | Real-time data stream processes to detect patterns, trends, and anomalies as they occur.                                                                                                               |
+|                           | Other (Analytics)                 | Analytics services that do not fall into one of the defined subcategories.                                                                                                                             |
+| Business Applications     | Other (Business Applications)     | All Business Applications services.                                                                                                                                                                    |
+| Compute                   | Containers                        | Management and orchestration of containerized compute platforms.                                                                                                                                       |
+|                           | End User Computing                | Virtualized desktop infrastructure and device / endpoint management.                                                                                                                                   |
+|                           | Quantum                           | Resources and simulators that leverage the principles of quantum mechanics.                                                                                                                            |
+|                           | Serverless Compute                | Enablement of compute capabilities without provisioning or managing servers.                                                                                                                           |
+|                           | Virtual Machines                  | Computing environments created by abstracting resources from physical hardware.                                                                                                                        |
+|                           | Other (Compute)                   | Compute services that do not fall into one of the defined subcategories.                                                                                                                               |
```

Comment:
@arrarama mentioned this as well above. 

@salpaysenturus's comment is that at in Azure, AWS have bare metal instances under EC2 and Azure VM. Therefore there's no single service that is bare metal only. Would it make sense to move to a 'Compute Instances' subcategory with a description like: 
'Computing instances created by abstracting resources from physical hardware or as direct .'

OR Compute Machine or Compute Server with a description like:
'Managed virtual or physical compute machines/servers'

Reply:
First off, I'll copy my reply to [@arrarama's bare-metal comment](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/pull/519#discussion_r1734997909) here:

> we focused on defining those subcategories that have the most commonality across all providers. In the specific case of bare metal, it could be misleading to maintain such a subcategory, given that it would not be inclusive of bare metal activity across all providers. For example, AWS bare metal instance types live under the EC2 service, which is categorized as `Virtual Machines`.
> 
> It is my belief that the definition of a _SKU_ taxonomy will handle for practitioner analyses such as these, and the Service taxonomy cannot provide that kind of fine grain.
> 
> As it stands, there would be no appropriate subcategory for OCI Bare Metal Servers, and so it would go into `Compute > Other (Compute)`. Please keep in mind that there are many, many services across all the providers that will not have a subcategory home in the first version of this taxonomy, as a guiding principle here is to undercategorize rather than overcategorize and allow ourselves (and the community) to advocate for adding missing subcategories over time.

Given that context (which Udam summarized above), I'm certainly open to changing the name of `Virtual Machines` to be inclusive of bare-metal.

How does `Provisioned Compute` feel?  I would want this to live alongside `Serverless Compute` to delineate between, say, EC2 vs Lambda.

Reply:
Discussed at length on the Sep 3 TF1 call; decided to keep bare-metal under `Virtual Machines` with no name change.  Resolving.